matlab/lib/+Ice/CloseConnectionException.m

```diff
@@ -7,24 +7,6 @@
 % propagated to the application code.
```

Comment:
We need to rework the copyyright and doc-comments in a follow-up PR. I believe it should be inside the classdef, not above.